src/sentry/middleware/subdomain.py

```diff
@@ -30,7 +31,10 @@ def __call__(self, request: Request) -> Response:
         if not self.base_hostname:
             return self.get_response(request)
 
-        host = request.get_host().lower()
+        try:
+            host = request.get_host().lower()
```

Comment:
should you still lowercase it if `get_host` fails?

Reply:
@JoshFerge If an exception is raised in `get_host()`, I expect the chained `lower()` method to not be called at all.

I'm unsure if I understood what you meant by here.

Reply:
ah needed to expand logic below to see how host was being used, that makes sense 👍🏼